2015/presentations/attaching_legs_to_a_snake/text.md

```diff
@@ -0,0 +1,582 @@
+# Attaching legs to a snake - or - Python3 extensions - Tomasz Maćkowiak
+
+## Introduction
+
+Python is a powerful language by itself. The robust standard library and the myriad of additional packages make it a Swiss Army knife of programming languages. But sometimes it is not enough. When you need to integrate with a low-level C API or execute heavy computations without the interpreter overhead, writing a Python extension module might be a good idea.
```

Comment:
The robust standard library and the myriad of additional packages make it a Swiss Army knife for any programming task.